catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -659,10 +720,30 @@ private static boolean isClassificationAdded(
         addCoverageNodeWithAttributes(
             graph, partNode, metacard, orb, attribute + ":", resultAttributes));
 
-    Attribute typeAttr = metacard.getAttribute(Core.DATATYPE);
-    if (typeAttr != null) {
-      type = getType(String.valueOf(typeAttr.getValue()));
-    }
+    /*
+       // DCMI type vocabulary labels
+    COLLECTION("Collection"), //
+    DATASET("Dataset"), //
+    EVENT("Event"), //
+    IMAGE("Image"), //
+    INTERACTIVE_RESOURCE("Interactive Resource"), //
+    MOVING_IMAGE("Moving Image"), //
+    PHYSICAL_OBJECT("Physical Object"), //
+    SERVICE("Service"), //
+    SOFTWARE("Software"), //
+    SOUND("Sound"), //
+    STILL_IMAGE("Still Image"), //
+    TEXT("Text");
+       */
+    /*    Attribute typeAttr = metacard.getAttribute(Core.DATATYPE);
+        if (typeAttr != null) {
+          type = getType(String.valueOf(typeAttr.getValue()));
+        }
+    */
+
+    // TODO: What's correct? typeConversionMap or NsiliProductType
+    // figure out what the NSILI product type should be
```

Comment:
There is a lot of commented out code. Is this to be implemented in the future and should it be removed until then?

Reply:
I was confused about this too. This previously used the NsiliProductType and the switch to typeConversionMap seems incomplete as most of it's commented.

@beyelerb What was the intent with the typeConversionMap? Does the mapping need more work or is it fine as is?

Reply:
The issue was that the type retrieved from the metacard doesn't match the type names from the NSIL model. In some cases, one DCMI type (text) is overloaded to represent distinct NSIL model types. This should be a future ticket to clean all this up and represent as a clean, logical method of translating between the two models.

Reply:
Created https://codice.atlassian.net/browse/CAL-520 to address DCMI <-> NSIL type mappings